benchmarks/experimental/offload.py

```diff
@@ -149,6 +150,10 @@ def train_epoch(args):
 
 
 def train(model_config, model, benchmark_config, model_specs, args):
+    device = torch.device("cuda")
+    torch.cuda.set_device(0)
+    torch.manual_seed(5)
```

Comment:
jus curious, why the 5 ?

Reply:
Just a number for me to compare weights. Removing this in a follow up PR where I add golden numbers for the benchmarks.